app/views/global_search/_products.html.haml

```diff
@@ -0,0 +1,20 @@
+%h3= Product.model_name.human(count: 2)
+
+%table.table.table-striped.table-hover.js--responsive_table
+  %thead
+    %tr
+      %th= "#{Product.model_name.human} Name"
+      %th= "Product Type"
+      %th= Facility.model_name.human
+      %th= Product.human_attribute_name("description")
+  %tbody
+    - results.each do |product|
+      %tr
+        %td.nowrap.span2
+          = link_to product.name + (product.is_hidden? ? ' (hidden)' : ''), facility_product_path(product.facility, product)
+        %td.span2
+          = product.model_name.human
+        %td.span2
+          = link_to product.facility, facility_path(product.facility)
+        %td.span6
+          = product.description
```

Comment:
My general feeling is we should not display the entire thing. For facilities we have a short description. We could also use the rails `truncate` helper.

Reply:
I kinda like displaying the whole thing, honestly, but am willing to be convinced otherwise. It looks okay to me with Dartmouth data.

If we were to truncate, what length should we truncate at? 300 characters?